test/fork/migration-v2.js

```diff
@@ -95,7 +141,7 @@ describe('v2 migration', function () {
   });
 
   // generates the eligibleForCLAUnlock.json file
-  it.skip('run get-products-v1 script', async function () {
+  it.skip('run get-locked-in-v1-claim-assessment script', async function () {
```

Comment:
agreed to resolve all solidity issues and contract deployment issues in this first PR.

Going back to these scripts to allow to fix migration issues after.